ssl/ssl_lib.c

```diff
@@ -1716,6 +1716,10 @@ int SSL_shutdown(SSL *s)
 
 int SSL_renegotiate(SSL *s)
 {
+    /* Do nothing in TLS1.3 */
```

Comment:
Well this is a public API function used to request a renegotiation - so it could happen. Possibly we could treat this as an error because it doesn't make sense...but I thought it was better just to ignore it. Another possibility is we could interpret this as a request to send a KeyUpdate in TLSv1.3 - but we haven't implemented that yet.